src/Domain/LeanCode.DomainModels/Model/TimestampTz.cs

```diff
@@ -0,0 +1,93 @@
+using System.Text.Json.Serialization;
+
+namespace LeanCode.DomainModels.Model;
+
+/// <remarks>
+/// This type is intended to be used with PostgreSQL which supports AT TIME ZONE operator
+/// with IANA time zone IDs but cannot store both timestamp and offset in a single column.
+/// </remarks>
+public sealed record class TimestampTz : ValueObject
+{
+    public static bool IsValidTimeZoneId(string timeZoneId)
```

Comment:
Nit: methods (no matter if static or not) should be placed after properties.

Reply:
Pls gib linter/analyzer for this :weary: